ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -619,12 +613,41 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
-    private func updateSheetExpandedContentHeight() {
-        if let bottomSheetController = bottomSheetController {
-            bottomSheetController.preferredExpandedContentHeight = estimatedTableViewHeight + Constants.BottomSheet.expandedContentTopMargin
+    /// Recalculates header top margin constraint and updates the `collapsedContentHeight` and `isExpandable` properties of the sheet controller.
+    private func updateSheetHeaderSizingParameters() {
+        guard let bottomSheetController = bottomSheetController else {
+            return
+        }
+        bottomSheetController.isExpandable = isExpandable
+
+        let maxHeroItemHeight = heroCommandStack.arrangedSubviews.map { $0.intrinsicContentSize.height }.max() ?? Constants.defaultHeroButtonHeight
+        let headerHeightWithoutBottomWhitespace = Constants.BottomSheet.headerTopMargin + maxHeroItemHeight
+
+        // How much more whitespace is required at the bottom of the sheet header
```

Comment:
padding might be a better word than whitespace, which is confusing already in dark mode?